Source/WebCore/html/EmailInputType.cpp

```diff
@@ -68,7 +68,7 @@ bool EmailInputType::typeMismatchFor(const String& value) const
     if (!element()->multiple())
         return !isValidEmailAddress(value);
     for (auto& address : value.splitAllowingEmptyEntries(',')) {
-        if (!isValidEmailAddress(stripLeadingAndTrailingHTMLSpaces(address)))
+        if (!isValidEmailAddress(address.trim(isASCIIWhitespace)))
```

Comment:
Would be better if this used `StringView`. Seems the string is not going to be stored.